Gemfile.lock

```diff
@@ -276,12 +276,12 @@ DEPENDENCIES
   selenium-webdriver
   sprockets-rails
   stimulus-rails
-  turbo-rails
+  turbo-rails (>= 2.0.6)
   tzinfo-data
   web-console
 
 RUBY VERSION
    ruby 3.2.2p53
 
 BUNDLED WITH
-   2.4.13
+   2.4.6
```

Comment:
can you update your bundler version so it's not changing each time someone adds a new gem?